packages/talisman-ui/src/components/MysticalBackgroundV3/MysticalPhysicsV3.ts

```diff
@@ -13,7 +13,7 @@ export type MysticalPhysicsV3 = {
 
 export const MYSTICAL_PHYSICS_V3: MysticalPhysicsV3 = {
   artifacts: 3,
-  blur: 24,
+  blur: 0,
```

Comment:
This actually didn't work before!
I've fixed it so that setting `blur` actually does something - but then I've set the default config to not use any blur - since we weren't using it and it looks good without it anyway (and it costs a bit to render it)